tools/local_fix_format.sh

```diff
@@ -22,20 +22,24 @@
 # If DISPLAY is set, then tkdiff pops up for some BUILD changes.
 unset DISPLAY
 
+# The following optional argument is added to be able to run this script using Docker,
+# due to a problem to locate clang using WSL on Windows. Call with -docker as the first arument.
+if [[ $# -gt 0 && "$1" == "-docker" ]]; then
+  exec ./ci/run_envoy_docker.sh "$0" -run-build-setup "$@"
```

Comment:
I think it would be cleaner to call `shift` before exec here to remove the `-docker` param, and only have one `shift` under `-run_build-setup`.

Reply:
done